src/app/loadout-drawer/loadout-utils.ts

```diff
@@ -226,13 +226,14 @@ export function getLoadoutStats(
   });
 
   // Add stats that come from the subclass fragments
-  if (subclass?.socketOverrides) {
-    for (const plugHash of Object.values(subclass.socketOverrides)) {
+  // TODO: Now that we apply socket overrides when we resolve items, do we need to do this calculation?
```

Comment:
Would the alternative be to pull the stats directly off the overridden subclass? 

I gave this a go by removing the calculation and replacing it with
```
for (const stat of subclass?.item.stats || []) {
    stats[stat.statHash].value += stat.value;
  }
```

It appears that the calculation differs from beta when doing this, note the recovery difference. I haven't investigated but I would wager that we aren't handling conditional mods correctly in one of the two.

### Beta
<img width="1203" alt="Screen Shot 2022-03-11 at 11 29 31 pm" src="https://user-images.githubusercontent.com/7344652/157877271-15157082-28e1-4bfc-b2cc-7403b1697505.png">

### Dev
<img width="1236" alt="Screen Shot 2022-03-11 at 11 29 47 pm" src="https://user-images.githubusercontent.com/7344652/157877423-08f600d6-d7b5-444c-a5ea-f040150cc301.png">

### Conditional subclass mod
<img width="269" alt="Screen Shot 2022-03-11 at 11 31 13 pm" src="https://user-images.githubusercontent.com/7344652/157877511-3926a9b8-e461-48fc-b68c-bdb55ea182df.png">

Reply:
Seems possible, and that's why I didn't remove that code. But what I'd like is for applying stat overrides to result in the correct stats, which would allow us to do that.